packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -282,7 +296,6 @@ class TabController extends ChangeNotifier {
   set offset(double value) {
     assert(value != null);
     assert(value >= -1.0 && value <= 1.0);
-    assert(!indexIsChanging);
```

Comment:
The gif that shows interruption of the animateTo looks worse than i expected. Both the page and tab bar have a big screw. I think we should probably make it uninterruptible.

Reply:
Ok, the only thing that changed in this PR was to make TabBar get interrupted. TabBarView was always getting interrupted because ```Scrollable``` widget inside TabBarView was cancelling the animation.
If we should make them all uninterruptible, I guess I should do something somewhere in Scrollable widget. I'll have a look at it.